docs/utils.py

```
@@ -126,3 +139,34 @@
 	text = re.sub(r'[^\w\- ]', "", s).strip().lower()
 	# Then replace consecutive spaces or dashes with a single dash
 	return re.sub(r'[-\s]+', "-", text)
+
+
+def sanitize_url(url):
+	"""ADD DOCS.
+	"""
+	parsed_url = urlparse(url)
+	# Validate scheme
+	if parsed_url.scheme not in URL_ALLOWED_SCHEMES:
+		raise ValueError("Invalid URL scheme. Only 'https' is allowed.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)